.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -68,7 +68,7 @@ This change added tests and can be verified as follows:
 
 ### Documentation
 
-Check the box below and label this PR (if you have committer privilege).
+Check the box below.
```

Comment:
```suggestion
Check the box below or label this PR directly (if you have committer privilege).
```

It is intended to mean "if you have committer privilege, you can label this PR directly rather than check the box in the PR description, which is more convenient for committers."
![image](https://user-images.githubusercontent.com/50226895/147548194-a11678c1-4b0e-49b9-a19b-cf1b9686b035.png)